assets/css/index.css

```diff
@@ -1,16 +1,126 @@
 :root {
   --background-color: #fff;
-  --primary-color: ;
-  --secondary-color: ;
+  --primary-color: #f15f36;
+  --secondary-color: #1fbbed;
 }
 
 * {
   font-family: 'Roboto', sans-serif;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  width: 100vw;
+  height: 100vh;
+}
+
+body {
+  background-color: var(--background-color);
+}
+
+.main-container {
+  width: 100vw;
+  height: 100vw;
+  object-fit: cover;
+}
+
+.search-container {
+  width: 100vw;
+  height: 80vh;
+  position: relative;
+}
+
+.main-image {
+  width: 100vw;
+  height: 80vh;
```

Comment:
It is really needed to use the viewport width and height in all of these places? I guess you have the width and height of the parent element (sai, the `search-container`), then it should be a matter of using a percentage of that size instead of the size of the viewport.